src/test/java/org/kiwiproject/registry/consul/config/ConsulRegistrationConfigTest.java

```diff
@@ -0,0 +1,19 @@
+package org.kiwiproject.registry.consul.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("ConsulRegistrationConfig")
+class ConsulRegistrationConfigTest {
+
+    @Test
+    void shouldDefaultProperties() {
```

Comment:
_nitpick:_ Suggest rename to `shouldHaveDefaultValues()` (I guess I prefer the test names to be a like statement...when I first read I had to think whether "Default"  was being used as a noun or as a verb.